rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/Doctrine/preserve_int_key_defined.php.inc

```diff
@@ -19,7 +22,7 @@ namespace Rector\Tests\Php80\Rector\Class_\AnnotationToAttributeRector\Fixture\D
 
 use Doctrine\ORM\Mapping as ORM;
 
-#[ORM\DiscriminatorMap([1 => 'CostDetailEntity'])]
+#[ORM\DiscriminatorMap(['cost' => 'CostEntity', 'product' => 'ProductEntity'])]
```

Comment:
"1" to 1 need to be kept to represent fixture to keep integer key unordered (direct 1)

Reply:
see the original PR why key 1 added here https://github.com/rectorphp/rector-src/pull/2735

Reply:
I checked this fixture and this is always `array<string, string>` list:
https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/inheritance-mapping.html#class-table-inheritance

Not sure how this got through :)

Reply:
the issue show the usage of key 1 directly https://github.com/rectorphp/rector/issues/7345

Reply:
I need to finish the feature first, then we'll deal with those edgecases. Otherwise I'm loosing focus and cannot progress.